### DISCUSSION TITLE
Update Node on Glitch-hosted examples

### DISCUSSION THREAD

**Discussion by shepazon** [mdn/mdn-community#663] [Content]
Back when I was working on [WebRTC](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling), [WebSocket](https://webrtc-from-chat.glitch.me/), and (IIRC) the WebGL examples, I needed to be able to host them somewhere that could handle the protocols involved. I wound up putting runnable and clonable versions of my examples on Glitch, and the documentation links to these examples.

The examples are not runnable at the moment because they have out-of-date Node versions specified. Someone needs to log in and update them. I no longer have access to the Glitch account I used while at Mozilla since it was tied to my no-longer-extant email address at `mozilla.com`.

The Node version update _should_ be the only thing needed to bring them back into working order. The only other option I can think of is to re-host the examples elsewhere.

If the examples can't be updated where they are, someone needs to move them to a new Glitch account and update MDN to point to the new location. This means changing both the link to try the examples in real time as well as the link to browse the code there.

-- Sheppy

**Comment by bsmth** [comment]
Hi Sheppy, great to hear from you & thanks a lot for letting us know 👋🏻

Just to check; the source for the examples is here:

- WebRTC
  - https://github.com/mdn/samples-server/tree/master/s/webrtc-from-chat
  - https://github.com/mdn/samples-server/tree/master/s/webrtc-simple-datachannel
  - etc.
- WebSockets: https://github.com/mdn/samples-server/tree/master/s/websocket-chat


>If the examples can't be updated where they are, someone needs to move them to a new Glitch account and update MDN to point to the new location.

I think this is what we have to do, we can check for glitch links and migrate them to a new account for now.

Also related: https://github.com/mdn/content/issues/4263

TY!
